surl/__init__.py

```diff
@@ -239,33 +224,56 @@ def get_config_from_cli(parser, auth_dir):
         except ConfigError:
             raise CliError(
                 "** Deprecated or Broken authentication file, "
-                "please delete it and login again:\n  $ rm {}".format(
-                    auth_path
-                )
+                "please delete it and login again:\n  $ rm {}".format(auth_path)
             )
 
         return config, remainder
 
+    # TODO: change this to use either snap or charm based on URL
```

Comment:
Educate me about the plans for this TODO. Is it covered by other JIRA tickets? Thanks!

Reply:
This is covered in https://warthogs.atlassian.net/browse/SN-1021